app/controllers/miq_capacity_controller.rb

```diff
@@ -210,8 +210,7 @@ def planning_option_changed
     end
     if params[:target_typ]
       @sb[:planning][:options][:target_typ] = params[:target_typ]
-      @sb[:planning][:options][:target_filters] = {}
-      MiqSearch.find_all_by_db(@sb[:planning][:options][:target_typ]).each { |ms| @sb[:planning][:options][:target_filters][ms.id.to_s] = ms.description }
+      @sb[:planning][:options][:target_filters] = MiqSearch.where(:id => params[:target_typ]).descriptions
       @sb[:planning][:options][:target_filter] = nil
```

Comment:
@chessbyte looking at this closer, that is the value that was there before.

I think singular holds the value that the use chose and plural holds the possible values